### PR TITLE
python3Packages.wasabi: 0.7.1 -> 0.8.0

### PR DESCRIPTION
**Pull request by danieldk**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by danieldk** [comment]
@ofborg build python37Packages.spacy python38Packages.spacy

**Review by drewrisinger — APPROVED** [review]
* [x] Diff mostly LGTM. Suggestions would be to add a changelog and switch to ``pytestCheckHook``, but not required.
* [x] Commits LGTM.
* [x] Builds via ``nix-review``

```
9 packages built:
python27Packages.wasabi python37Packages.spacy python37Packages.textacy python37Packages.thinc python37Packages.wasabi python38Packages.spacy python38Packages.textacy python38Packages.thinc python38Packages.wasabi
```

**Comment by danieldk** [comment]
>     * [x]  Diff mostly LGTM. Suggestions would be to add a changelog and switch to `pytestCheckHook`, but not required.

Version bumps are always a good excuse to add other improvements, so switched to `pytestCheckHook` and added `meta.changelog`. The commit message now also has a pointer to the changelog item.

Thanks for the comments!

File: pkgs/development/python-modules/wasabi/default.nix [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
   };
 
-  checkInputs = [
-    pytest
-  ];
-
-  checkPhase = ''
-    pytest wasabi/tests
-  '';
+  nativeBuildInputs = [ pytestCheckHook ];
```
Comment (drewrisinger) [inline pkgs/development/python-modules/wasabi/default.nix:16]:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

Comment (danieldk) [reply pkgs/development/python-modules/wasabi/default.nix]: Whoops, not sure how I missed that. :fearful:

**Review by drewrisinger — APPROVED** [review]
* [x] Diff LGTM
* [x] Commits LGTM
* [x] Builds via ``nix-review``:

```
https://github.com/NixOS/nixpkgs/pull/96464
9 packages built:
python27Packages.wasabi python37Packages.spacy python37Packages.textacy python37Packages.thinc python37Packages.wasabi python38Packages.spacy python38Packages.textacy python38Packages.thinc python38Packages.wasabi
```

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 96464 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>9 packages built:</summary><br>- python27Packages.wasabi<br>- python37Packages.spacy<br>- python37Packages.textacy<br>- python37Packages.thinc<br>- python37Packages.wasabi<br>- python38Packages.spacy<br>- python38Packages.textacy<br>- python38Packages.thinc<br>- python38Packages.wasabi</details>

**Pull request merged**